x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -123,6 +125,32 @@ const nonAggregatableDatasetsRoute = createDatasetQualityServerRoute({
   },
 });
 
+const degradedFieldsRoute = createDatasetQualityServerRoute({
```

Comment:
**doubt**: where is the versioning of this API handled? Given the [specs for having versioned APIs](https://docs.elastic.dev/kibana-dev-docs/versioning-http-apis) in all the environments (+ guides for [versioning interfaces](https://docs.elastic.dev/kibana-dev-docs/versioning-interfaces)) I can't figure out looking at server/client communication if the versioning is in place. If is missing, could you add it to the endpoint to comply with the specs please?

Reply:
We don't version internal endpoints. Also looking at the specs, its not mandatory to do.
Do you forsee any challenges ?